### ISSUE TITLE
The "show all columns starting from column n" (~[n-]) grep is broken.

### ISSUE THREAD

**Issue by pyllyukko** [radare/radare2#9123]
### Work environment

| Questions                                | Answers
|------------------------------------------|--------------------
| OS/arch/bits                             | Slackware x86 64
| r2 -v full output                        | radare2 2.2.0 0 @ linux-x86-64 git.2.2.0
| r2 -V full output                        | 2.2.0 all the way

### Expected behavior

Print columns starting from the spesified column, e.g.:

```
[0x00000000]> wx 10203040
[0x00000000]> px~:1[1-]
1020 3040 0000 0000 0000 0000 0000 0000 . 0@............
```

### Actual behavior

```
[0x00000000]> wx 10203040
[0x00000000]> px~:1[1-]
1020
```

### Steps to reproduce the behavior 

<https://github.com/pyllyukko/radare2-regressions/commit/cb320dc423ef219321c51c106a708a223d089c78>

**Comment by Maijin** [comment]
Can you make a PR for this test please ?

**Comment by pyllyukko** [comment]
> Can you make a PR for this test please,

Will do. I just wanted some confirmation to this issue before doing the PR.

> that's not the latest r2 from git btw please update.

Same applies with:


| Questions                                | Answers
|------------------------------------------|--------------------
| OS/arch/bits                             | Debian 9 x86 64
| r2 -v full output                        | radare2 2.3.0-git 16985 @ linux-x86-64 git.2.2.0-101-g8526f3393
| r2 -V full output                        | 2.2.0-101-g8526f3393

**Comment by zadlg** [comment]
Strange, `px~:1[1- ]` actually works. The problem comes from the `-` which is right before `]`. I'll try to fix that

**Comment by pyllyukko** [comment]
> Strange, px~:1[1- ] actually works.

For me ```px~:1[1- ]``` also prints column ```0```, which it shouldn't.

**Comment by zadlg** [comment]
Another bug.
I've fixed both.

**Comment by sivaramaaa** [comment]
Nice @zadlg !!! , also from next PR use reference like  "Fix #9123" to  your title or commit message , to automatically close the issue :)  (you have added 'issue' in between )
